src/AWActionParser.js

```diff
@@ -237,16 +249,41 @@ function resolveCommand(commandName, commandArguments) {
     return command;
 }
 
-function resolveIncompleteCoordinates(coordinates) {
+function resolveIncompleteCoordinates(coordinates, isScale) {
+  const clamp = (num, min, max) => Math.min(Math.max(num, min), max);
+  let [minScale, maxScale] = [0.1, 10];
+
+  let [x,y,z] = [coordinates[0], coordinates[1], coordinates[2]];
+
+  if (isScale) {
```

Comment:
It may may make more sense to have two functions instead doing everything in one single `resolveIncompleteCoordinates` and split the whole logic with a big if-else statement.
Maybe have something like `resolveIncompleteScaleCoordinates` separately, idk.